### PR TITLE
Speed up destroy_links for subject sets

### LINKED ISSUE #974: destroy_links is slow
similar to https://github.com/zooniverse/Panoptes/issues/672

Same issue as update_links, a destroy on each as well as any subsequeunt count callbacks, etc. Instead we should bulk delete the `set_member_subjects`, the associated `subject_workflow_counts` and manually queue the QueueRemovalWorker. 

Once the `set_member_subjects` are taken care of we can then remove the `subject_set` and it's `subject_set_workflows` by a normal delete probably (taking care to reset the subject_set#set_member_subject relation).

### LINKED ISSUE #987: sending transcriptions/classifications/markings directly to Cassandra
@rogerhutchings and @VVH would like to run a beta of annoTate starting July 1st. I'd like to be able to read in the transcriptions directly from Cassandra. @camallen thinks this is doable

### PR DESCRIPTION
**Pull request by edpaget**
Delete links without callbacks and manually queue workers. Closes #974

### CODE REVIEW

**Comment by camallen** [comment]
@ed - can you update the issue that this PR closes? Should be #974

**Comment by edpaget** [comment]
Sorry should be fixed.

File: spec/controllers/api/v1/subject_sets_controller_spec.rb [hunk 1]
```diff
@@ -205,4 +205,38 @@
 
     it_behaves_like "is destructable"
   end
+
+  describe '#destroy_links', :focus do
```
Comment (camallen) [inline spec/controllers/api/v1/subject_sets_controller_spec.rb]: Can you remove the focus here?

Comment (edpaget) [inline spec/controllers/api/v1/subject_sets_controller_spec.rb]: I was wondering why travis ran so fast.

**Pull request merged**